packages/kbn-expandable-flyout/src/components/preview_section.test.tsx

```diff
@@ -16,18 +16,26 @@ import {
   PREVIEW_SECTION_TEST_ID,
 } from './test_ids';
 import { TestProvider } from '../test/provider';
-import { State } from '../state';
+import { State } from '../store/state';
 
 describe('PreviewSection', () => {
-  const context = {
-    right: {},
-    left: {},
-    preview: [
-      {
-        id: 'key',
+  const context: State = {
+    data: {
+      panels: {
+        byId: {
```

Comment:
what do you think about combining `panels` and `byId` into one:
```
data: {
   byId:{
      flyout: {
         ...
      },
   needSync: true
```
it just seems we don't intend to put anything in`panels` layer other than by ids? this could simplify the reducers just a bit :)